app/presenters/step_by_step_page_presenter.rb

```diff
@@ -57,6 +57,14 @@ def summary_list_params
     params
   end
 
+  def broken_link_status_text(step)
```

Comment:
This feels like this is in the wrong place. It feels like this should be on the step, no the step-by-step-page.